beanstalk.go

```diff
@@ -95,12 +95,16 @@ func ValidURIs(uris []string) error {
 	}
 
 	for _, uri := range uris {
-		hostport, _, err := parseURI(uri)
+		address, uriType, err := parseURI(uri)
```

Comment:
Brilliant! This should be merged

Reply:
Fine! But only because you said so, not because @cooldarkdryplace made the PR 😉